app/modules/sightings/models.py

```diff
@@ -1310,11 +1363,15 @@ def _get_attempts_for_config(self, config_id, algorithm_id, annotation_guid_str)
         else:
             return 0
 
-    def _has_active_jobs(self, algorithm_id, annotation_guid_str):
+    def _has_active_jobs(self, config_id, algorithm_id, annotation_guid_str):
         for job in self.jobs:
             if (
-                self.jobs[job]['algorithm'] == algorithm_id
+                self.jobs[job]['algorithm']
```

Comment:
this was a bug as it was comparing algorithm name with algorithm id which never matched.....